### PR TITLE
Decompiled ovl_En_Ma3

### PR DESCRIPTION
**Pull request by ethteck**
Also tidied a function in Ma1 (I must have messed up when I tried that the first time)

### CODE REVIEW

File: src/overlays/actors/ovl_En_Ma3/z_en_ma3.c [hunk 1]
```diff
@@ -209,7 +199,7 @@ u16 func_80AA2AA0(GlobalContext *globalCtx, Actor *this) {
             break;
```
Comment (JoshDuMan) [inline src/overlays/actors/ovl_En_Ma3/z_en_ma3.c:199]: Have you tried removing all of these cases? Seems a bit unneeded.

Comment (ethteck) [reply src/overlays/actors/ovl_En_Ma3/z_en_ma3.c:199]: It looks like I can get away with only doing cases 0 and 9, but I thought it was more explicit and clear to list them all.

**Comment by Roman971** [comment]
Any way this could be merged/rebased so it doesn't also contain the diff from Ma2? Makes it hard to review as is.

**Comment by ethteck** [comment]
Yeah I agree, the diff sucks. I'll see what I can do. I may need to manually re-create it, but it shouldn't be too hard.

**Comment by ethteck** [comment]
Recreated as #114

**Pull request closed without merging**